apps/web/app/s/[surveyId]/page.tsx

```diff
@@ -192,6 +193,7 @@ export default async function LinkSurveyPage({ params, searchParams }: LinkSurve
       singleUseId={isSingleUseSurvey ? singleUseId : undefined}
       singleUseResponse={singleUseResponse ? singleUseResponse : undefined}
       webAppUrl={WEBAPP_URL}
+      responseCount={responseCount}
```

Comment:
As this is passing the `responseCount` to the client, please do this conditionally
```
survey.welcomeCard.showResponseCount ? responseCount : undefined
```